win32port/win32helpers/gettimeofday.c

```diff
@@ -1,5 +1,5 @@
 #include <time.h>
-#include <windows.h> //I've omitted context line

```

Comment:
Avoided breakage of build for normal Windows by cleaning up duplicate content in gettimeofday.c, removing header inclusions from gettimeofday.h and fixing include order in test-echo.c, test-ping.c and test-server.c